source/windows/bcrypt_ecc.c

```diff
@@ -470,3 +470,10 @@ struct aws_ecc_key_pair *aws_ecc_key_pair_new_from_asn1(
     }
     return NULL;
 }
+
+int aws_ecc_key_pair_append_asn1_encoding(struct aws_ecc_key_pair *key_pair, struct aws_byte_buf *buffer) {
+    (void)key_pair;
+    (void)buffer;
+
+    return AWS_OP_ERR;
+}
```

Comment:
Newline.... just an FYI... you prolly just want BCRYPTExportKey when the time comes.

Reply:
Started down that route, it only seems to export in the internal windows format which isn't helpful.  Likely we'd need to just do the full DER encoding ourselves which would remove the platform nonsense but the time needed for impl/debugging did not seem worth it right now.